src/libraries/Common/src/Interop/Unix/System.Native/Interop.UTimensat.cs

```diff
@@ -24,5 +24,8 @@ internal struct TimeSpec
         /// </returns>
         [GeneratedDllImport(Libraries.SystemNative, EntryPoint = "SystemNative_UTimensat", CharSet = CharSet.Ansi, SetLastError = true)]
         internal static unsafe partial int UTimensat(string path, TimeSpec* times);
+
+        [GeneratedDllImport(Libraries.SystemNative, EntryPoint = "SystemNative_FUTimens", CharSet = CharSet.Ansi, SetLastError = true)]
```

Comment:
@jkoritzinsky just curious, why does the generator not error when CharSet is specified and there are no strings involved? Seems like the opportunity to enforce this tidiness is the release we add the generator.

Reply:
We haven't added additional validation that checks if the `CharSet` property is specified but unneeded. I think that would be worth adding to improve the experience.